File: Demo3/app/src/main/java/com/hiscene/demo3/eventbus/UserInfo.java
package com.hiscene.demo3.eventbus;

import androidx.annotation.NonNull;

/**
 * Created by hujun on 2019-10-31.
 */

public class UserInfo {
    String name;
    int age;

    public UserInfo(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    @NonNull
    @Override
    public String toString() {
        return "name:"+name+",age="+age;
    }
}

File: Demo2/architecturecomponent/src/main/java/com/hiscene/architecturecomponent/UserInfo.kt
package com.hiscene.architecturecomponent

import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * Created by junhu on 2019-11-04
 */
@Entity
data class UserInfo(
        @PrimaryKey
        var id: Int = 0,
        var name: String = "",
        var age: Int = 0
)